diff --git a/node_modules/@cpelements/pfe-navigation/dist/pfe-navigation.js b/node_modules/@cpelements/pfe-navigation/dist/pfe-navigation.js
index ae679b8..9863bc0 100644
--- a/node_modules/@cpelements/pfe-navigation/dist/pfe-navigation.js
+++ b/node_modules/@cpelements/pfe-navigation/dist/pfe-navigation.js
@@ -1,6 +1,6 @@
-import PFElement from '../../../@patternfly/pfelement/dist/pfelement.js';
-import '../../../@patternfly/pfe-icon/dist/pfe-icon.js';
-import '../../../@patternfly/pfe-avatar/dist/pfe-avatar.js';
+import PFElement from '@patternfly/pfelement/dist/pfelement.js';
+import '@patternfly/pfe-icon';
+import '@patternfly/pfe-avatar/dist/pfe-avatar.js';
 
 /*!
  * PatternFly Elements: PfeNavigation 1.0.106
